Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN apt-get update \
 WORKDIR /tmp
 
 ## build the esbUtils (not directly available as a jar so build and install locally).
-RUN git clone --branch v2.1 https://github.com/tl-its-umich-edu/esbUtils \
+RUN git clone --branch v2.2 https://github.com/dlhaines/esbUtils \
```

Comment:
this needs to be updated back to official repo rather than personal branch

Reply:
+1